gcp/appengine/frontend3/src/templates/list.html

```diff
@@ -55,128 +52,133 @@ <h1 class="title">Vulnerability Library</h1>
+            </span>
+            <span role="cell" class="vuln-table-cell vuln-packages mdc-data-table__cell">
+              <ul class="packages">
+                {%- if vulnerability.affected | map(attribute='package', default=[]) | list == [{}] -%}
+                {%- for repo in vulnerability.affected | git_repo | map('strip_scheme') -%}
+                {%- if loop.index < 8 -%} <li>{{ repo }}</li>
+                  {%- elif loop.index == 8 -%}
                   <li>...</li>
-                {%- endif -%}
+                  {%- endif -%}
+                  {%- else -%}
+                  <li>See details.</li>
+                  {%- endfor -%}
+                  {%- else -%}
+                  {%- for affected in vulnerability.affected -%}
+                  <li>{{ affected.package.ecosystem }}/{{ affected.package.name }}</li>
+                  {%- endfor -%}
+                  {%- endif -%}
+              </ul>
+            </span>
+            <span role="cell" class="vuln-table-cell vuln-summary mdc-data-table__cell hide-on-mobile">
+              {{ vulnerability.summary or "See record for full details" }}
+            </span>
+            <span role="cell" class="vuln-table-cell vuln-versions mdc-data-table__cell hide-on-mobile">
+              <ul class="versions">
+                {#-
+                https://stackoverflow.com/questions/31876069/is-it-possible-to-flatten-a-lists-of-lists-with-ansible-jinja2
+                -#}
+                {%- for version in vulnerability.affected | map(attribute='versions', default=[]) | sum(start=[]) -%}
+                {% if loop.index < 8 %} <li class="version">{{ version }}</li>
+                  {%- elif loop.index == 8 -%}
+                  <li class="version">...</li>
+                  {%- endif -%}
+                  {%- else -%}
+                  <li class="version">See details.</li>
+                  {%- endfor -%}
+              </ul>
+            </span>
+            <span role="cell" class="vuln-table-cell mdc-data-table__cell hide-on-mobile">
+              <relative-time datetime="{{ vulnerability.modified }}">
+                {{ vulnerability.modified }}
+              </relative-time>
+            </span>
+            <span role="cell" class="vuln-table-cell vuln-fix-status mdc-data-table__cell hide-on-mobile">
+              {%- if vulnerability.isFixed -%}
+              <span class="tag fix-available">Fix available</span>
               {%- else -%}
-                <li>See details.</li>
-              {%- endfor -%}
-            {%- else -%}
-            {%- for affected in vulnerability.affected -%}
-              <li>{{ affected.package.ecosystem }}/{{ affected.package.name }}</li>
-            {%- endfor -%}
-            {%- endif -%}
-            </ul>
-          </span>
-          <span role="cell" class="vuln-table-cell vuln-summary mdc-data-table__cell hide-on-mobile">
-            {{ vulnerability.summary or "Not available, see record for full details" }}
```

Comment:
FYI this is the only line that _actually_ changed. VS Code formatting on save is a bit special.